app-modules/student-data-model/src/Filament/Tables/StudentsTable.php

```diff
@@ -102,6 +102,9 @@ public function __invoke(Table $table): Table
                             ->label('SAP')
                             ->icon('heroicon-m-academic-cap'),
                         BooleanConstraint::make('dual'),
+                        BooleanConstraint::make('firstgen')
+                            ->label('First Generation')
+                            ->icon('heroicon-m-lock-open'),
```

Comment:
Please check with Product to confirm whether or not this is the icon that should be used.

Reply:
Icon updated as Product suggest in ticket in jira.